src/config/2021.json

```diff
@@ -364,6 +364,15 @@
       "github": "demianrenzulli",
       "twitter": "drenzulli"
     },
+    "dominiclovell": {
+      "name": "Dominic Lovell",
+      "teams": [
+        "authors",
+        "analysts"
+      ],
+      "avatar_url": "https://avatars.githubusercontent.com/u/131131?v=4&s=200",
+      "github": "dominiclovell"
+    },
```

Comment:
@dominiclovell FYI adding your contributor info. Open a new PR to make any changes.